packages/core/src/routes/sign-in-experience.ts

```diff
@@ -0,0 +1,44 @@
+import { SignInExperiences } from '@logto/schemas';
+import { object, string } from 'zod';
+
+import koaGuard from '@/middleware/koa-guard';
+import {
+  findDefaultSignInExperience,
+  updateSignInExperienceById,
+} from '@/queries/sign-in-experience';
+
+import { AuthedRouter } from './types';
+
+export default function SignInExperiencesRoutes<T extends AuthedRouter>(router: T) {
+  /**
+   * As we only support single signInExperience settings for V1
+   * always return the default settings in DB for the /sign-in-ex get method
+   */
+  router.get('/sign-in-ex', async (ctx, next) => {
+    const SignInExperience = await findDefaultSignInExperience();
```

Comment:
should we use camel-case instead of pascal-case for local variable `SignInExperience`?

Reply:
good call updated